src/utils/apiCalls.ts

```diff
@@ -1,4 +1,4 @@
-export const fetchGetResponse = async (endPoint: string) => {
-  let url = `https://search.ams.usda.gov/farmersmarkets/v1/data.svc/${endPoint}`
+export const fetchData = async (endPoint: string) => {
```

Comment:
While we aren't doing POST or DELETE for this project my issue with this naming convention is that POST and DELETE would require separate functions and both technically Fetch data. Is there a reason to change the specificity here?

Reply:
Ahhh I see, that makes sense. I can change to getData ? I just thought it was overly wordy and confusing